gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/cockpit/command/handler/OrganizationCommandHandler.java

```diff
@@ -63,7 +67,7 @@ public OrganizationCommandHandler(OrganizationService organizationService) {
             newOrganization.setDomainRestrictions(organizationPayload.getDomainRestrictions());
 
             final OrganizationEntity organization = organizationService.createOrUpdate(
-                GraviteeContext.getExecutionContext(),
+                executionContext,
```

Comment:
Is there a point to keep both parameters - executionContext - and organiszationId, in OrganizationService.createOrUpdate ? As execution context contains the organizationId ?

Reply:
As you wish. I thought it would keep the consistency on all services, context + resourceId 
but 🤷‍♂️